components/Forms/Select/Select.tsx

```diff
@@ -1,5 +1,5 @@
 import React, { FC, useCallback, useEffect, useState } from 'react';
-import ReactSelect, { SelectProps as ReactSelectProps, components } from 'react-select';
+import ReactSelect, { Props as ReactSelectProps, components } from 'react-select';
```

Comment:
For reference: [index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e58b370380ea5595a59ef2f8144629253855e06d/types/react-select/index.d.ts#L22) -> [src/Select.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/e58b370380ea5595a59ef2f8144629253855e06d/types/react-select/src/Select.d.ts#L56)